tests/ui/parser/unnecessary-let.stderr

```diff
@@ -2,7 +2,7 @@ error: expected pattern, found `let`
   --> $DIR/unnecessary-let.rs:2:9
    |
 LL |     for let x of [1, 2, 3] {}
-   |         ^^^^
+   |         ^^^
```

Comment:
Can you add

```
//@ run-rustfix
```

to `tests/ui/parser/unnecessary-let.rs` then rebless that test, so we can see what the *after* of the suggestion being applied looks like?

Reply:
🫡